### PR TITLE
Override golang 1.14.4 and associated bbclasses, no folder changes necessary

### PR DESCRIPTION
**Pull request by x85446**
golang 1.14.4 recipe and associated bbclasses is taken from the master branch of yocto at this point in time: https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?id=ca701cb92dcf186664d9b1f82c679e4f9cc388ec

### CODE REVIEW

**Comment by x85446** [comment]
Confirmed: Since we don't have CI/CD on this, I can confirm golang 14.4 ended up on the build.   Job #30, Artifact 117

> root@raspberrypi3:~# go version
go version go1.14.4 linux/arm

File: docs/tips_tricks_hacks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Tips Tricks and Hacks
```
Comment (ygoyal18) [inline docs/tips_tricks_hacks.md:1]: @x85446 As this is not related to this PR, so we should remove it and bring it in through another PR.

**Comment by ygoyal18** [comment]
@x85446 Apart from above comment, it LGTM. If we have used any reference to bring in these files, it will be good to mention it in the PR.

File: docs/tips_tricks_hacks.md (hunk 1 above)
Comment (x85446) [reply docs/tips_tricks_hacks.md]: removed with subsequent commit.

**Comment by x85446** [comment]
> @x85446 Apart from above comment, it LGTM. If we have used any reference to bring in these files, it will be good to mention it in the PR.

Great, I have dropped the file and referenced the yocto Sha in the PR.

golang 1.14.4 recipe and associated bbclasses is taken from the master branch of yocto at this point in time: https://git.yoctoproject.org/cgit/cgit.cgi/poky/commit/?id=ca701cb92dcf186664d9b1f82c679e4f9cc388ec
